app/models/hyrax/permission_template.rb

```diff
@@ -225,7 +222,7 @@ def read_groups
     #   visibility when applying permission template ACLs
     # @return [Boolean]
     def reset_access_controls(interpret_visibility: false)
-      reset_access_controls_for(collection: source_model,
+      reset_access_controls_for(collection: source,
```

Comment:
this shouldn't change, i think. it was implemented to ensure the implementation for `reset_access_controls` stayed the same for AF models, after the refactor.

Reply:
I changed this back to use `source_model`.